Classes/Controller/LoginController.php

```diff
@@ -79,7 +79,9 @@ public function login(string $_, ?array $pluginConfiguration)
             // performRedirectAfterLogin stops flow by emitting a redirect
             $this->performRedirectAfterLogin();
         }
-        $this->performRedirectToLogin($pluginConfiguration['authorizationUrlOptions.']);
+        $this->performRedirectToLogin(
+            isset($pluginConfiguration['authorizationUrlOptions.'])  ? $pluginConfiguration['authorizationUrlOptions.']: []
```

Comment:
PSR-12 / CGL, simplify to `$pluginConfiguration['authorizationUrlOptions.'] ?? []`

Reply:
Done